<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Comment:</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ECG Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RESP Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>IBP Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SPO2 Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMP Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ARR Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wave Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Para Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NIBP DIS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start
NIBP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Module
Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SelectgroupBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Select:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>72</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Module type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelComPort">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>COM port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBaudrate">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>BM100A</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxComPort">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBaudrate">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxNibp">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>24</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>NIBP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>24</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>mmHg</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>24</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>成人</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>24</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>静止</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>135</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0/0 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>31</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Press</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>31</y>
       <width>18</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>51</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pluse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>51</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>31</y>
       <width>18</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PR:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>31</y>
       <width>18</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_12</zorder>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>72</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataNumValue">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
